<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestTask</class>
 <widget class="QMainWindow" name="RequestTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
    background: qradialgradient(
        cx: 0.5, cy: 0.5,
        fx: 0.5, fy: 0.5,
        radius: 0.5,
        stop: 0.2981 rgba(241, 166, 97, 255),
        stop: 1 rgba(227, 139, 41, 255)
    );
}
QFrame {
    background: transparent;
    border: none;
}
QLineEdit{
	color: rgb(0, 0, 0,0.6);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>410</width>
           <height>500</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="request">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FDEEDC;</string>
            </property>
            <property name="text">
             <string>Request Task</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QFrame" name="frame_2">
            <property name="styleSheet">
             <string notr="true">QLineEdit,QComboBox,QDateTimeEdit {
    height: 50px;
    border-radius: 5px;
    border: 1px solid #fff;
    padding-left: 10px;
    background-color: #FDEEDC;
	color:#E38B29;
}

QLabel{
	background-color: #FDEEDC;
color:#E38B29;
    border-radius: 5px;
}

QComboBox::drop-down,QComboBox::down-arrow  {
	margin:20px;
}

QPushButton{
	border: 1px solid #FFD8A9;
    height: 20px;
    background-color: #FFD8A9;
border-radius: 20px;
color:#E38B29;
}

QPushButton:hover{
    background-color: #9c6644;
border: 1px solid #9c6644;
	color:#FFD8A9;
transition: all 0.3s ease-in-out;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="topMargin">
              <number>9</number>
             </property>
             <item row="1" column="0" colspan="2">
              <widget class="QComboBox" name="comboBox">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="placeholderText">
                <string>Select Task...</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QPushButton" name="add">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Preferred Time Slots</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
               <property name="indent">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDateTimeEdit" name="preferredTimeSlots"/>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QLineEdit" name="description">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
	height: 120px;
	border-radius: 10px;
	border: 1px solid #fff;
	padding-left:10px;
	background-color: #FDEEDC;

border-radius: 15px;
}</string>
               </property>
               <property name="placeholderText">
                <string>Request Description</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QPushButton" name="cancel">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLineEdit" name="taskName">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Task Name</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: 1px solid #FFD8A9;
    height: 20px;
    background-color: #FFD8A9;
border-radius: 20px;
color:#E38B29;
}

QPushButton:hover{
    background-color: #9c6644;
border: 1px solid #9c6644;
	color:#FFD8A9;
transition: all 0.3s ease-in-out;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="back">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
